projects/binutils/fuzz_readelf_non_persistent.c

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2022 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "readelf.h"
```

Comment:
We have this because of the oss-fuzz set up (https://github.com/google/oss-fuzz/blob/4c109322213e64ab5f09a6d0f1ae39c56b6f0ad2/projects/binutils/build.sh#L82) so the `readelf.h` file hols all of the `readelf.c` code.